### PR TITLE
Request to merge "HTML: SIGN UP PAGE" to master #2

### PR DESCRIPTION
**Pull request by RailonA**
We made a clone of the New York Times page. The page works best on a 1366x768 resolution and using the firefox browser, assets might look out of place using other browsers or resolutions. The page was build using grid, flex, and float to align all of the elements properly.

Project Milestones:

- Use proper selector elements

- Implement grid and flex displays

- Float property to position the aside section

### CODE REVIEW

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @RailonA and @realisticattorney  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
### :warning: However we need to make few changes to mark this project approved. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [ ] Remove the horizontal scrolbar
- [ ] Give some spacing between each box in suggestted newsletter for you section, and make sure each box has button with circled border
- [ ] Make sure the footer looks like the original, the very last footer shouldbe hrizontal, but yours is vertical
- [ ] All a tags should have href attributes
- [ ] alt attributes on images are all empty :point_right: that is a bad practice, you should give them a short descriptive texts. Example: `alt="erc's profile"` 
- [ ] The middle newyork time logo in navbar should be a link.
- [ ] Decrease text size of the footer of main text and increase the width of the `READ 615 COMMENTS` button
- [ ] Remove the `signup` button which is appearing in `Suggested newsletters for you`
- [ ] Don't commit files which aren't related to the project, :point_right: Remove the `.vscode` directory from your project
- [ ] Use the `.stylelintrc.json` file from microverse on [this link](https://github.com/microverseinc/linters-config/blob/master/html-css/.stylelintrc.json)
- [ ] You are repeating yourself in your CSS codes, please revise it again and eliminate all repeated codes, take advange of using classes, or grouping similar elements
- [ ] Implement all changes I suggested in inline comments in your codes

# [Optional changes] :large_blue_circle: 
- [ ] Text color should change when the sbscribe and login buttons are hovered in the navbar
- [ ] Most popular article should change the text color (shade of grey) when hovered
- [ ] Your commit messages should be consistently in imperative mode or present tense, you should have used `Fix footer grid` instead of `Fixed footer grid`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,918 @@
+<!DOCTYPE html>
+<html lang="en">
+
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (jubaan) [inline index.html:10]: - [ ] Keep  code DRY (Don't Repeat Yourself), delete one of the repeated code.

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

> * [x] Remove the horizontal scrolbar
- [ ]  Give some spacing between each box in suggestted newsletter for you section, and make sure each box has button with circled border
**The boxes in the original page contain a margin between each box, the projects don't, please fix this before submitting.**
- [ ]  Make sure the footer looks like the original, the very last footer shouldbe hrizontal, but yours is vertical
**The footer still doesn't look like the original, also the NYT logo should be a link**
image
- [ ]  All a tags should have href attributes
**Not all `<a>` anchor tags have `href` attribute**
> * [x]  alt attributes on images are all empty point_right that is a bad practice, you should give them a short descriptive texts. Example: `alt="erc's profile"`
> * [x]  The middle newyork time logo in navbar should be a link.
> * [x]  Decrease text size of the footer of main text and increase the width of the `READ 615 COMMENTS` button
> * [x]  Remove the `signup` button which is appearing in `Suggested newsletters for you`
> * [x]  Don't commit files which aren't related to the project, point_right Remove the `.vscode` directory from your project
> * [x]  Use the `.stylelintrc.json` file from microverse on [this link](https://github.com/microverseinc/linters-config/blob/master/html-css/.stylelintrc.json)

### [OPTIONAL]  :yellow_circle:  

- * [ ]  You are repeating yourself in your CSS codes, please revise it again and eliminate all repeated codes, take advange of using classes, or grouping similar elements
- * [ ]  Implement all changes I suggested in inline comments in your codes

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required
**Hi @RailonA & @jubaan**
great job guys, you have done really well with the requirements and the TSE's changes.
there is a problem though, the image in the middle of the page is not working.
please fix the image before submitting another request.
I really wanted to approve, but that image has to be working.

if you need any further clarification, please contact me on slack `Mahmoud Mohammad`

**Review by umairarshadbutt — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
- The cloned page is rendering as it should and meets all the project specifications.
- The project has a descriptive README.md and is professionally done. Good job with that.
- Pull Request has a meaningful title and a short description.
- CSS is clean. No linter errors at this point and no bad practices used as well.
- The markup as well has no errors nor warnings to show. Good job with that.
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job realisticattorney! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
**If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Pull request merged**